core/clipboard.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * noVNC: HTML5 VNC client
+ * Copyright (c) 2025 Tobias Fahleson
+ * Licensed under MPL 2.0 or any later version (see LICENSE.txt)
+ */
+
+import * as Log from './util/logging.js';
+
+export default class Clipboard {
+    constructor(target) {
+        this._target = target;
+        this._isGrabbed = false;
+        this.onRead = () => {};
```

Comment:
This is old style event handling. Please use the helper class we have to get the modern style.